data/human/far north jobs.txt

```diff
@@ -86,15 +86,17 @@ mission "Navy resupply [0]"
 	cargo Military 5 2 .1
 	job
 	repeat
-	clearance
+	deadline
 	to offer
 		random < 2
 	source
-		not government Pirate
+		government "Republic" "Syndicate"
 		attributes outfitter factory
 	destination
-		attributes north military
-		attributes outfitter
+		government "Republic"
+		distance 2 10
+		attributes north
+		attributes military outfitter
```

Comment:
```suggestion
		attributes military outfitter
		not attributes rich
```
With this filter added, the possible destinations should be:
- Farpoint
- Prime
- Triton Station (no clearance actually means you can't land here)
- Maelstrom

So we need to add "station" to that list, too.
```suggestion
		attributes military outfitter
		not attributes rich station
```

Reply:
I don't think location filter chooses planets you can't land on unless you use `ignore clearance`. I could be wrong though.